src/routes/sessions/[id]/+page.svelte

```diff
@@ -169,12 +172,24 @@
 <div class="session">
 	<Header>
 		<p data-testid="session-id" class="text-sm font-bold leading-none">
-			Session <Button size="link" variant="link" href={`/${session.id}`}>#{session.id}</Button>
+			Session <Button size="link" variant="link" href={`/sessions/${session.id}`}
+				>#{session.id}</Button
+			>
 		</p>
-		<p data-testid="model-name" class="text-sm text-muted">
-			{isNewSession ? 'New session' : session.model}
-		</p>
-
+		<div class="grid grid-cols-[auto,auto] gap-x-1">
+			{#if isNewSession}
+				<p data-testid="new-session-text" class="text-sm text-muted">New session</p>
+			{:else}
+				{#if session.updatedAt}
+					<Badge variant="default" capitalize={false}>
+						{formatShortDistanceToNow(new Date(session.updatedAt), { addSuffix: true })}
+					</Badge>
+				{/if}
+				<Badge variant="default" capitalize={false}>
+					<p data-testid="model-name">{session.model}</p>
+				</Badge>
+			{/if}
+		</div>
```

Comment:
Not a huge fan of how this turned out visually (not your fault).
I need to play with the design a bit more.

In the mean time, let's revert back to the old `<p>` component:

```svelte
<p data-testid="model-name" class="text-sm text-muted">
   {isNewSession ? 'New session' : session.model}
</p>
```